src/GraphQL.Tests/Types/GraphTypesLookupTests.cs

```diff
@@ -1,38 +0,0 @@
-using System.Threading;
-using System.Threading.Tasks;
-using GraphQL.Types;
-using Xunit;
-
-namespace GraphQL.Tests.Types
-{
-    public class GraphTypesLookupTests
-    {
-        private class TestType : GraphType { }
-
-        private readonly ManualResetEvent _inIteration = new ManualResetEvent(false);
-        private readonly ManualResetEvent _lookupModified = new ManualResetEvent(false);
-        private readonly GraphTypesLookup _lookup = new GraphTypesLookup();
-
-        [Fact]
-        public void is_thread_safe()
```

Comment:
This test doesn't make sense anymore because Lookup doesn't allow you to change its content after the schema is created.